octobot_trading/exchanges/connectors/ccxt/ccxt_websocket_connector.py

```diff
@@ -632,6 +632,8 @@ async def _wait_for_initialization(self, feed, *g_args, **g_kwargs):
         is_initialized_func = None
         if feed is Feeds.CANDLE:
             def candle_is_initialized_func():
+                if self.exchange_manager is None:
+                    return False
```

Comment:
Should we raise instead of returning a value?

Reply:
this happens in tests, it's just to avoid raising attribute errror